cdx_writer/dispatcher.py

```diff
@@ -87,5 +94,12 @@ def dispatch_resource(self, record, env):
     def dispatch_warcinfo(self, record, env):
         return WarcinfoHandler
 
+    def dispatch_metadata(self, record, env):
+        content_type = record.content_type
+
+        if content_type and content_type.startswith('application/json;generator-youtube-dl'):
```

Comment:
mimetype is `application/json;generator=youtube-dl`?

Reply:
I am open to making anything you want. I thought it captured what we needed.  Suppose we save some other type of metadata from something not youtube-dl.  We could then change the identifier to "generator-different-app" making it easily identifiable for parsing.  But that is my assumption with knowing all the details of the playback code.